controllers/dockercluster_controller.go

```diff
@@ -77,9 +79,32 @@ func (r *DockerClusterReconciler) Reconcile(req ctrl.Request) (ctrl.Result, erro
 		return ctrl.Result{}, err
 	}
 
+	dockerCluster.Status.Ready = true
```

Comment:
this needs to move up to before the patch otherwise we'll be saving it and *then* updating it and not saving it.

Reply:
Fixed.